tests/07-simulation-base/28-ipv6-tcp-sockets.csc

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<simconf>
+  <project EXPORT="discard">[APPS_DIR]/mspsim</project>
```

Comment:
These two project lines are not required (and should not be generated by the latest Cooja).

Reply:
The "project" lines have been removed now.